CHANGELOG.md

```diff
@@ -1,6 +1,11 @@
 CHANGELOG
 =========
 
+# 4.4.0 / 2020-09-30
+
+* [BUGFIX] (Windows) Fix compatibility with Ansible 2.10. See [#289][].
+* [FEATURE] Adds support for 3rd party integrations via the `datadog-agent integration` command. See [#291][].
```

Comment:
https://github.com/DataDog/ansible-datadog/pull/297 is a minor fix, but I think it should be mentioned nonetheless.